docs/src/app/components/pages/components/GridList/ExampleComplex.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import GridList from 'material-ui/lib/grid-list/grid-list';
+import GridTile from 'material-ui/lib/grid-list/grid-tile';
+import StarBorder from 'material-ui/svg-icons/toggle/star-border';
+import IconButton from 'icon-button';
+
+const tilesData = [
+  {
+    img: 'images/grid-list/00-52-29-429_640.jpg',
+    title: 'Breakfast',
+    author: 'jill111',
+    featured: true,
+  },
+  {
+    img: 'images/grid-list/burger-827309_640.jpg',
+    title: 'Tasty burger',
+    author: 'pashminu',
+  },
+  {
+    img: 'images/grid-list/camera-813814_640.jpg',
+    title: 'Camera',
+    author: 'Danson67',
+  },
+  {
+    img: 'images/grid-list/morning-819362_640.jpg',
+    title: 'Morning',
+    author: 'fancycrave1',
+    featured: true,
+  },
+  {
+    img: 'images/grid-list/hats-829509_640.jpg',
+    title: 'Hats',
+    author: 'Hans',
+  },
+  {
+    img: 'images/grid-list/honey-823614_640.jpg',
+    title: 'Honey',
+    author: 'fancycravel',
+  },
+  {
+    img: 'images/grid-list/vegetables-790022_640.jpg',
+    title: 'Vegetables',
+    author: 'jill111',
+  },
+  {
+    img: 'images/grid-list/water-plant-821293_640.jpg',
+    title: 'Water plant',
+    author: 'BkrmadtyaKarki',
+  },
+];
+
+const gradientBg = 'linear-gradient(to bottom, rgba(0,0,0,0.7) 0%,rgba(0,0,0,0.3) 70%,rgba(0,0,0,0) 100%)';
+
+export default class GridListExampleSimple extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div style={{display: 'flex', flexWrap: 'wrap', justifyContent: 'space-around'}}>
+        {/*Grid list with all possible overrides*/}
+        <GridList
+          cols={2}
+          cellHeight={200}
+          padding={1}
+          style={{width: 500, height: 400, overflowY: 'auto', marginBottom: 24}}
+          >
+          {
+            tilesData.map(tile => <GridTile
```

Comment:
Took me a while to get to bottom of this -_-

This little guy breaks code highlighting. Take it out into a variable and all works well:

```js
const titleElements = tilesData.map(tile => <GridTile
              key={tile.img}
              title={tile.title}
              actionIcon={<IconButton><StarBorder color="white"/></IconButton>}
              actionPosition="left"
              titlePosition="top"
              titleBackground={gradientBg}
              cols={tile.featured ? 2 : 1}
              rows={tile.featured ? 2 : 1}
              ><img src={tile.img} /></GridTile>);

// ....
        <GridList
          cols={2}
          cellHeight={200}
          padding={1}
          style={{width: 500, height: 400, overflowY: 'auto', marginBottom: 24}}
          >
          {titleElements}
        </GridList>
// ....

```